### DISCUSSION TITLE
Hilfe-Fenster hängt Oben in der Ecke

### DISCUSSION THREAD

**Discussion by RysingTV** [LSS-Manager/LSSM-V.4#2037] [Q&A] [answered]
Hi, habe seit kurzem das das Hilfe-Fenster oben Rechts in der Ecke hängt und ich bekomme dies nicht mehr verschoben. 
Kennt oder hat da jemand eine Lösung? 

![image](https://user-images.githubusercontent.com/39877792/209438105-6af2e0c2-aa37-4c79-bb57-3fd8322483f6.png)

Ich habe es schon komplett Rausgenommen, Browser vollständig gelöscht und nochmal neu Installiert und und und und... weis leider keine Lösung außer zu Fragen.

**Comment by jxn-30** [comment, marked as answer]
Das passiert leider manchmal.
Beheben lässt sich das wie folgt:
1. Einsatz in einem neuen Tab öffnen
2. Dort die Konsole (F12 -> Konsole) öffnen
3. Den folgenden Code ausführen:
```js
lssmv4.$stores.settings.setSetting({moduleId: 'missionHelper', settingId: 'overlay', value: false}).then(() => lssmv4.$stores.settings.setSetting({moduleId: 'missionHelper', settingId: 'drag', value: {active: false, top: 50, left: 50}}));
```
Das setzt einerseits die Position zurück und verlässt auch den angepinnten Modus.
Sobald das nächste mal ein Einsatz geöffnet wird, sollte wieder alles gut sein.

**Reply by RysingTV** [reply]
Oh vielen Dank, 

alleine das öffnen im neuen Tab hat mich schon das Fenster verschieben lassen. 
Eine einfache Lösung mit Großer Wirkung.
